### DISCUSSION TITLE
Appimage o Flatpak package

### DISCUSSION THREAD

**Discussion by martincalveira** [helix-editor/helix#1968] [Ideas]
Hi!

Is it on the roadmap to create an Appimage or Flatpak package? 

Thank you.

**Comment by martincalveira** [comment]
In Ubuntu 20.04 I have not been able to install it by compiling, that is why I was asking if it was available in these forms of distribution.

**Reply by the-mikedavis** [reply]
The AppImage part was added and should be included in the next release: https://github.com/helix-editor/helix/issues/2076

**Comment by martincalveira** [comment]
Thank you!
